frontend/packages/console-shared/src/components/dashboard/status-card/AlertItem.tsx

```diff
@@ -39,15 +40,19 @@ export const StatusItem: React.FC<StatusItemProps> = ({ Icon, timestamp, message
   );
 };
 
-const AlertItem: React.FC<AlertItemProps> = ({ alert }) => (
-  <StatusItem
-    Icon={getSeverityIcon(getAlertSeverity(alert))}
-    timestamp={getAlertTime(alert)}
-    message={getAlertDescription(alert) || getAlertMessage(alert)}
-  >
-    <Link to={alertURL(alert, alert.rule.id)}>View details</Link>
-  </StatusItem>
-);
+const AlertItem: React.FC<AlertItemProps> = ({ alert }) => {
+  const actionAlertMatch = actionAlerts.find((actionAlert) => actionAlert.name === alert.rule.name);
```

Comment:
I'd just call this `action`.

```suggestion
  const action = actionAlerts.find((actionAlert) => actionAlert.name === alert.rule.name);
```

Reply:
fixed